content/docs/performance-and-scalability/realistic-app-benchmark.md

```diff
@@ -1,9 +1,84 @@
 ---
-title: Realistic Application Benchmark
+title: Customer-like Application Benchmark
 description: Performance measurement through realistic micro service application tests.
 weight: 40
 ---
 
-For realistic application benchmarks we use [IBM's BluePerf](https://github.com/blueperf).
+The Regression Patrol for Istio Performance is an automated suite of tests running a customer-like microservices application (Blueperf, a.k.a, Acmeair) on an IBM Cloud Kubernetes Service (IKS) cluster using the latest available Istio build as the service mesh orchestrator. 
+
+Its objective is to detect performance regression in the current build and provide data to possibly identify the reason for the regression.  We selected the maximum throughput provided by the system as the single Key Performance Indicator (KPI) for evaluating whether the performance has regressed compared to previous builds.
+
+Three configurations are tested for each build:
+
+**•	Istio Full:** Istio is deployed to the K8s cluster using the manifest file that comes in the build, with the default configuration. Some changes are made to force pods to certain nodes.
+
+**•	No Mixer:** the Telemetry and Policy functions of Mixer are disabled in order to isolate their impact on the overall performance.
+
+**•	Ingress Only:** the application is re-deployed without sidecars, so basically just the Istio Ingress is being exercised. This provides an approximate baseline where Istio is not used.
+
+The regression patrol perform these main tasks:
+1.	Checks for new builds in the [Istio Daily Build Repository](https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/ );
```

Comment:
on the preview this didn't turn into a bullet list

Reply:
if a numbered list is desired use
```
1. item one 
1. item two
1. item thre
```

or a bulleted list
```
- item 1
- item 2
- item 3
```

The problem here is probably the extra spaces, coupled with the numbers not always starting with 1.